charts/kubeclarity/values.yaml

```diff
@@ -341,3 +343,19 @@ kubeclarity-postgresql:
 
 # End of KubeClarity Postgres Values
 #######################################################################################
+
+#######################################################################################
+## KubeClarity External Postgres Values
+
+# notes: self deployed postgresql and external-postgresql can't be used at the same time
+# use this if you wan't to use an external postgresql database as your backend 
```

Comment:
I think it would be a good idea to add something like this into the template for safety:
```
{{- if .Values.kubeclarity-postgresql.enabled }}
{{- if .Values.kubeclarity-postgresql-external.enabled }}
{{- fail "kubeclarity-postgresql and kubeclarity-postgresql-external can not both be enabled!" }}
{{- end }}
{{- end }}
```

Reply:
I added this at the start of deployment.yaml, do you think this is the right place to put this ? Or should those safety check be elsewhere ?

Reply:
I think that makes sense for now, perhaps if we have more in the future we'll create a _validation file like the _helpers one